libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -1066,14 +1060,13 @@ export const appStrings = {
 
   noteScannerUndervoteContestsCardPlural: () => (
     <UiString uiStringKey="noteScannerUndervoteContestsCardPlural">
-      All other votes in these contests will count, even if you leave some
-      blank.
+      All other votes in this contest will count.
```

Comment:
Did you intentionally make this not plural anymore?

Reply:
No, got a fixup in the chamber.